src/oc_erchef/rebar.lock

```diff
@@ -68,7 +68,7 @@
  {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
  {<<"ibrowse">>,
   {git,"git://github.com/chef/ibrowse.git",
-       {ref,"1df08a6aca30614f69126451957af6a3d0aeb7b2"}},
+       {ref,"c4e319e193410e8e7527c5f43563c46a0718bca8"}},
```

Comment:
Lets keep on the same version for bookshelf and bifrost.

Reply:
bifrost is not using it and Bookshelf doesn’t have the direct usage of ibrowse except some test bookshelf_bench.erl which is pulling the ibrowse from `git://github.com/cmullaparthi/ibrowse.git`.
Please let me know If I still need to pass this flag from ominibus to bookshelf.
![Screenshot 2021-11-16 at 11 20 15 AM](https://user-images.githubusercontent.com/76031665/141930308-e31f8bad-fb63-4639-bb90-a809ef05c87e.png)

Reply:
Now that the changes for ibrowse are merged to main, should we use that merge commit hash?

Reply:
yeah this needs to be the hash in the chef/ibrowse repo with the corresponding changes for now.